shared/logutil/logutil.go

```diff
@@ -23,3 +26,19 @@ func ConfigurePersistentLogging(logFileName string) error {
 	logrus.Info("File logging initialized")
 	return nil
 }
+
+// CountdownToGenesis logs the time remains until genesis time
+func CountdownToGenesis(genesisTime time.Time, secondsCount int) {
```

Comment:
Is there a better name you could choose than `secondsCount`? 
It's not immediately clear what this variable is until the reader looks at the usage of  it.

Reply:
So I am going to rename it to seconds

Reply:
What about using a time.Duration? Does it have to be seconds?

Reply:
I changed it to duration